docs/data-sources/integration_idp.md

```diff
@@ -3,12 +3,12 @@
 page_title: "cyral_integration_idp Data Source - cyral"
 subcategory: ""
 description: |-
-  Retrieve and filter IdP integrations.
+  ~> DEPRECATED Use resource and data source cyral_integration_idp_saml instead.
 ---
 
 # cyral_integration_idp (Data Source)
 
-Retrieve and filter IdP integrations.
+~> **DEPRECATED** Use resource and data source `cyral_integration_idp_saml` instead.
```

Comment:
If we want to display this nicely with a warning box on the docs page, we need to add an extra new line before the deprecated message. This also applies to the other resource/datasource docs. If you want to check how it would look you can use this [doc preview tool](https://registry.terraform.io/tools/doc-preview).
```suggestion

~> **DEPRECATED** Use resource and data source `cyral_integration_idp_saml` instead.
```

Reply:
But it does look good already as there is a new line above it. Maybe you got confused with the code that shows in `description` field in the metadata section.

Reply:
I just pushed a change to piece of code that adds these deprecation warnings to avoid this from happening at any time in the future even if the docs template misses this line. Anyway, no changes in the commented docs as it is already ok as I mentioned previously.